src/xml_errors.cc

```diff
@@ -24,9 +24,7 @@ void set_numeric_field(Local<Object> obj, const char *name, const int value) {
 Local<Value> BuildSyntaxError(xmlError *error) {
   Nan::EscapableHandleScope scope;
 
-  Local<Value> err =
-      Exception::Error(Nan::New<String>(error->message).ToLocalChecked());
-  Local<Object> out = Local<Object>::Cast(err);
```

Comment:
The reason I removed this, because casting err to the out object is the reason the formatting for JS broke.

A stack trace doesn't fit nicely into a key value JS object.  Im trying to think of a more elegant way to append a stack trace.  Will update if I come up with anything.

Reply:
Also error->message might not be a stack trace, as im using that as xsd validation error messages.